### PR TITLE
Remove codecs folder from Plex config

### PR DESCRIPTION
**Pull request by mouzzampk2014**
Hi, 

Plex sometimes fails to play media and gives this error/message `required codecs cannot be found`. This normally happens after the update but this script written by iXNyNe fix this issue. 

This script runs every time plex container starts or update.

Branch is `plex-codecs`

Would be great if this one can be added as a MOD. 

I just forked my first repository this morning and never worked on github previously (except raising issues) so if I missed anything I apologies in advance.

Thanks

### CODE REVIEW

**Comment by Roxedus** [comment]
Please number the `plex-remove-codecs` file

**Comment by mouzzampk2014** [comment]
> Please number the `plex-remove-codecs` file

Hi Rox, all done. Thanks

**Comment by Roxedus** [comment]
Branch added, please rebase to it 👍

**Comment by mouzzampk2014** [comment]
> Branch added, please rebase to it 👍

Hi Rox, apologise for late reply :( I think I missed the notification. You know where I can find Rebase option? 

Apologise again

**Comment by Roxedus** [comment]
https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/changing-the-base-branch-of-a-pull-request

**Comment by mouzzampk2014** [comment]
> https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/changing-the-base-branch-of-a-pull-request

Thanks Rox. All done. Ta

File: .github/workflows/BuildImage.yml [hunk 1]
```diff
@@ -4,8 +4,8 @@ on: [push, pull_request, workflow_dispatch]
 
 env:
   ENDPOINT: "linuxserver/mods" #don't modify
-  BASEIMAGE: "replace_baseimage" #replace
-  MODNAME: "replace_modname" #replace
+  BASEIMAGE: "plex" #replace
+  MODNAME: "codecs" #replace
```
Comment (Roxedus) [inline .github/workflows/BuildImage.yml:8]: Should rename the mod to `remove-codecs`

**Review by aptalca — CHANGES_REQUESTED** [review]
Branch name doesn't match the mod name

**Comment by mouzzampk2014** [comment]
Thank you aptalca. I changed the branch name. Ta

**Comment by mouzzampk2014** [comment]
Oops apologise I think I done mistake after changing the branch name I created new pull request. Sorry not sure how to merge it.

**Comment by mouzzampk2014** [comment]
Sorry guys you know what I need to do. I mistakenly created new pull request when I supposed to change the mod name. Is it possible to open this and close the other one or shall I leave the other one open? Thank you

**Pull request closed without merging**